README.md

```diff
@@ -34,7 +34,20 @@ Do note however that Network.framework requires macOS 10.14+, iOS 12+, or tvOS 1
 
 ## Versioning
 
-This repository will remain in development throughout the beta period of macOS Mojave and iOS 12. Once those operating systems have gone GM, we will finalize the API and cut a stable release.
+Just like the rest of the SwiftNIO family, swift-nio-transport-services follows [SemVer 2.0.0](https://semver.org/#semantic-versioning-200) with a separate document
+declaring [SwiftNIO's Public API](https://github.com/apple/swift-nio/blob/master/docs/public-api.md).
+
+### `swift-nio-transport-services ` 1.x
+
+`swift-nio-http2` versions 1.x are a pure-Swift implementatation of the HTTP/2 protocol for SwiftNIO. It's part of the SwiftNIO 2 family of repositories and does not have any dependencies besides [`swift-nio`](https://github.com/apple/swift-nio), Swift 5, and an Apple OS supporting `Network.framework`. As the latest version, it lives on the [`master`](https://github.com/apple/swift-nio-transport-services) branch.
+
+To depend on `swift-nio-transport-services `, put the following in the `dependencies` of your `Package.swift`:
+
+    .package(url: "https://github.com/apple/swift-nio-transport-services.git", from: "1.0.0"),
+
+### `swift-nio-transport-services ` 0.x
+
+The legacy `swift-nio-transport-services` 0.x is part of the SwiftNIO 1 family of repositories and works on Swift 4.1 and newer but requires [nghttp2](https://nghttp2.org) to be installed on your system. The source code can be found on the [`swift-nio-transport-services-swift-4-maintenance `](https://github.com/apple/swift-nio-http2/tree/nghttp2-support-branch) branch.
```

Comment:
This is not true, it doesn't need nghttp2. Neither is the URL correct here.

Reply:
oops, thank you, fixed